dsl/camel-jbang/camel-jbang-core/src/test/java/org/apache/camel/dsl/jbang/core/commands/InitTest.java

```diff
@@ -17,18 +17,21 @@
 package org.apache.camel.dsl.jbang.core.commands;
 
 import java.io.File;
+import java.io.FileInputStream;
 import java.io.IOException;
+import java.nio.charset.Charset;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.List;
 
 import org.apache.camel.util.FileUtil;
+import org.apache.commons.io.IOUtils;
```

Comment:
Can we avoid using 3rd party libraries - use IOHelper from Camel or Files.readAllLines